src/pyfreeleh/row/query_builder.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any, List, Optional, Tuple
+
+from .base import Ordering
+
+
+class InvalidQuery(Exception):
+    pass
+
+
+class GoogleSheetQueryBuilder:
+    def __init__(self) -> None:
+        self._where: Optional[Tuple[str, Tuple[Any, ...]]] = None
+        self._orderings: List[Ordering] = []
+        self._limit: int = 0
+        self._offset: int = 0
+
+    def where(self, condition: str, *args: Any) -> "GoogleSheetQueryBuilder":
+        self._validate_where(condition, args)
+        self._where = (condition, args)
+        return self
+
+    def _validate_where(self, cond: str, args: Any) -> None:
+        if cond.count("?") != len(args):
+            raise InvalidQuery("number of placeholder and argument is not equal")
+
+    def _build_where(self) -> str:
+        # ASSUMPTION: the first field (A) is always the PK field.
+        query = "WHERE A IS NOT NULL"
```

Comment:
Why do we need this? I think this condition should be needed only for the UPDATE operation, right?

Reply:
I think it can also beneficial to all statements that has `WHERE` clause, such as `DELETE` and `COUNT` as well.